pkg/kubelet/stats/cadvisor_stats_provider.go

```diff
@@ -156,6 +156,9 @@ func (p *cadvisorStatsProvider) ListPodStats(ctx context.Context) ([]statsapi.Po
 			podStats.CPU = cpu
 			podStats.Memory = memory
 			podStats.Swap = cadvisorInfoToSwapStats(podInfo)
+			if utilfeature.DefaultFeatureGate.Enabled(features.KubeletPSI) {
+				podStats.IO = cadvisorInfoToIOStats(podInfo)
```

Comment:
Do we want to update the unit tests for changes inside stats/?

Reply:
@ndixita Added. PTAL

Reply:
Thank you